substrate-processor/src/processor/batchProcessor.ts

```diff
@@ -539,15 +539,10 @@ export class SubstrateBatchProcessor<Item extends {kind: string, name: string} =
     }
 
     /**
-     * Sets the maximum number of blocks which can be fetched
-     * from the data source in a single request.
-     *
-     * The default is 100.
+     * deprecated
```

Comment:
Add @ to deprecated - it will be parsed and strikethrough-highlighted by most IDE in user code
```
/**
@deprecated
*/
```